index.js

```diff
@@ -0,0 +1,39 @@
+const nav = document.getElementsByClassName('nav');
+const mobileNav = document.getElementsByClassName('mobile-nav');
+const humberger = document.getElementById('humberger');
+const close = document.getElementById('closemenu');
+const porfolioLink = document.getElementById('portfolioLink');
+const aboutLink = document.getElementById('aboutLink');
+const contactLink = document.getElementById('contactLink');
+
+porfolioLink.addEventListener('click', () => {
+  mobileNav[0].classList.remove('open');
+  nav[0].classList.remove('close');
+  mobileNav[0].classList.add('close');
+  nav[0].classList.add('openFlex');
+});
+aboutLink.addEventListener('click', () => {
+  mobileNav[0].classList.remove('open');
+  nav[0].classList.remove('close');
+  mobileNav[0].classList.add('close');
+  nav[0].classList.add('openFlex');
+});
+contactLink.addEventListener('click', () => {
+  mobileNav[0].classList.remove('open');
+  nav[0].classList.remove('close');
+  mobileNav[0].classList.add('close');
+  nav[0].classList.add('openFlex');
+});
+
+humberger.addEventListener('click', () => {
+  mobileNav[0].classList.remove('close');
+  nav[0].classList.remove('openFlex');
+  mobileNav[0].classList.add('open');
+  nav[0].classList.add('close');
+});
+close.addEventListener('click', () => {
+  mobileNav[0].classList.remove('open');
+  nav[0].classList.remove('close');
+  mobileNav[0].classList.add('close');
+  nav[0].classList.add('openFlex');
```

Comment:
- In order to follow Javascript best practices (DRY, YAGNI, KISS), I advise that you make all repeated blocks of code into a function and call the function when you need it, this will help keep your code clean and make it easy to maintain. Please fix this for all blocks of code that are repeated more than once 👍 👍